data_store/transformation/towns_fund/tf_transform_r3.py

```diff
@@ -716,7 +716,7 @@ def extract_risks(
     :param programme_id: ID of the programme for this ingest
     :return: A new DataFrame containing the extracted programme/risk rows.
     """
-    df_risk_programme = df_risk.iloc[9:14, 2:-1]
+    df_risk_programme = df_risk.iloc[9:14, 2:16]
```

Comment:
If this is only changing for round 6 then consider that this would break reingest for round 3, 4, and 5?

Reply:
Surely the table has the same length... right?!

Reply:
You tell me please 🙏  (ideally with evidence)

Reply:
Checked the spreadsheets and the Risk table all stretch up to column P. Also tests passed for those rounds too.

Reply:
Yeh. Maybe I was too cocky about my fix. 

Updating the tests revealed I missed updating a function. :disappointed: